package.json

```
@@ -5,6 +5,7 @@
   ],
   "scripts": {
     "dev": "yarn workspace @okta/vuepress-site dev",
+    "dev-redesign": "REDESIGN=true yarn run dev",
```

Minor nitpick but could we do:
```suggestion
    "dev:redesign": "REDESIGN=true yarn run dev",
```

:)